$OpenBSD: patch-data_evince_desktop_in_in,v 1.2 2008/12/14 19:33:48 ajacoutot Exp $

- We want to see the evince application in the menu.
- The Office menu is more appropriate.

--- data/evince.desktop.in.in.orig	Fri Dec 12 17:29:54 2008
+++ data/evince.desktop.in.in	Fri Dec 12 17:30:08 2008
@@ -8,7 +8,7 @@ StartupNotify=true
 Terminal=false
 Type=Application
 Icon=evince
-NoDisplay=true
+NoDisplay=false
 X-GNOME-DocPath=
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=evince
